### ISSUE TITLE
Solve deprecated class ProgressDialogue and NotificationCompat.Builder

### ISSUE THREAD

**Issue by 0xamogh** [openMF/android-client#1123]
These are the few deprecated classes which are still being used. I propose to replace them with their newer, improved versions for the good of the project.
Both the above classes are deprecated since API 26.

I am working on this issue

**Comment by harshitbansal05** [comment]
@amogh-jrules the one for `NotificationComapt.Builder` has already been opened by me. It just has a small problem of style template. I would fix it shortly, Thanks

**Comment by amogh-jrules** [comment]
Actually, I saw your issue before putting up this request. Are you refering to #1109? Your request is reference to deprecation of builder.sslSocketFactory of OkHttpClient class, whereas this references to NotificationCompat.Builder which is a builder class for Notification Objects. Please check into it :)

**Comment by harshitbansal05** [comment]
No, I am referring to #1103. I just have to add a style template to it, Thanks.

**Comment by amogh-jrules** [comment]
Oh okay! My bad, you can carry on with the PR, I will fix the remainder of this issue.
Thanks for letting me know! :)
